src-v5/new/carousel.tsx

```diff
@@ -191,119 +206,72 @@ const Carousel = (props: CarouselProps): React.ReactElement => {
   };
 
   const onTouchEnd = () => {
-    if (!props.dragging) {
+    handleDragEnd();
```

Comment:
should we remove onTouchEnd function at all and use handleDragEnd function. One function less

Reply:
True could move it within an inline function.